.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+test.md
```

Comment:
![image](https://user-images.githubusercontent.com/99715640/213889821-38879083-7c10-4c7e-a82f-5153c46f6e1b.png)


- [ ] As the previous reviewer asked that you please remove `A test.md` from your committed/pushed file directories, it appears you forgot. Please kindly do this. Removing unnecessary files like you did with `node_module` is a conventional practice that keeps our code base clean. To make sure this works, kindly update the `A test.html` file name to `test.md`; then in your git bash you can:
```
git rm A \test.md --cached
```

Reply:
- [ ] Good job working on your project so far :muscle: you're almost there. However, I noticed that you didn't remove the `A test.md` file from your pushed file directories as the previous reviewers suggested. I suggest that you kindly do this so your project can be approved. You can also try to fix it this way:
- [ ] Delete the `A test.md` file in your local repository and push the changes to github. 
- [ ] After this you create a new file in your local repo called `test.md`
- [ ] Then you push those changes to GitHub as well. :+1: 
- [ ]  This should have removed the previous file and created a new file with the same name as what was in the project requirements and what you have in the `.gitignore` file. Because of this, the new `test.md` file will not be pushed to GitHub but it will be ignored by git :100: then you can resubmit for review